lib/logstash/filters/mutate.rb

```diff
@@ -4,7 +4,7 @@
 
 # The mutate filter allows you to perform general mutations on fields. You
 # can rename, remove, replace, and modify fields in your events.
-class LogStash::Filters::Mutate < LogStash::Filters::Base
+module LogStash module Filters class Mutate < LogStash::Filters::Base
```

Comment:
Can this inherit from `< Base` instead? Seems like we could do that now since we're scoping our classes this way.